tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -76,6 +76,10 @@
   if mac_target and (pkg_resources.parse_version(mac_target) <
 		     pkg_resources.parse_version('10.9.0')):
     os.environ['MACOSX_DEPLOYMENT_TARGET'] = '10.9'
+    os.environ['_PYTHON_HOST_PLATFORM'] = re.sub(
```

Comment:
Use of `re` without import of `re`; smells like what's at the heart of `NameError: name 're' is not defined` in the failed test logs?